### ISSUE TITLE
Zombie update / Reanimation serum rework

### ISSUE THREAD

**Issue by MaxeUA** [studio-minus/ppg-bugs-and-requests#1185]
**Feature description**
-Reanimation spreads slower over the body, so if the limb cuts off, human won't recieve zombie effect
-Turning into a zombie will become slower, human will be weak, untill he fully becomes a zombie
-Improvised firearm: simmiliar to a flintlock pistol, but different
-A pump shotgun: wooden version of a normal one, reloads longer, and haves its own animation and particle effects
-A tool to cut off limbs, some kind of a hand-powered saw

**Feature purpose**
Much of the realism, will bring alot of fun into the game, will fix the unplayable zombies, and so on

**Additional context**
The walking dead..?

**Comment by CUBE0R** [comment]
> The reanimation serum rework, improvised firearm, pump shotgun and tool to cut off limbs are separate requests. Also, I'm pretty sure the tool to cut off limbs is basically a specialized chainsaw (since you can already cut off limbs with the chainsaw, it just takes some fiddling).
> 
> Not against it, just telling you that zooi probably cannot "use" this issue unless he intends to add everything mentioned.
 Agreed, should probably be in different requests, but good feature ideas. Chainsaw is way too clunky for proper use as its currently implemented, though. Especially for anything precise.

**Comment by DindinYT37** [comment]
> > The reanimation serum rework, improvised firearm, pump shotgun and tool to cut off limbs are separate requests. Also, I'm pretty sure the tool to cut off limbs is basically a specialized chainsaw (since you can already cut off limbs with the chainsaw, it just takes some fiddling).
> > Not against it, just telling you that zooi probably cannot "use" this issue unless he intends to add everything mentioned.
> > Agreed, should probably be in different requests, but good feature ideas. Chainsaw is way too clunky for proper use as its currently implemented, though. Especially for anything precise.

You uh... put your reply in the quote.

**Comment by TamasB14** [comment]
DindinYT37 said it right. I will summerize what he said simpler:
*"Do not group-request"*

**Comment by CUBE0R** [comment]
> My bad

**Comment by CUBE0R** [comment]
> > > The reanimation serum rework, improvised firearm, pump shotgun and tool to cut off limbs are separate requests. Also, I'm pretty sure the tool to cut off limbs is basically a specialized chainsaw (since you can already cut off limbs with the chainsaw, it just takes some fiddling).
> > > Not against it, just telling you that zooi probably cannot "use" this issue unless he intends to add everything mentioned.
> > > Agreed, should probably be in different requests, but good feature ideas. Chainsaw is way too clunky for proper use as its currently implemented, though. Especially for anything precise.
> 
> You uh... put your reply in the quote.

Yeah, fixed it.
